src/components/TableModule/types.ts

```diff
@@ -39,7 +39,7 @@ export interface TableCell<Item = any> extends TableAlignOptions {
   className?: string;
 }
 
-export interface TableConfiguration<Item extends RowSelectionRow> {
+export interface TableConfiguration<Item = any> {
```

Comment:
Making the default to be `any` seems too accommodating IMHO. I feel like if you explicitly set `any` yourself, then that's on you, but if the _default_ is `any` then it's on us that's promoting less than ideal typing. What do you think?

Reply:
I do want to make the type required. however, this PR is about backward compatibility to keep the type requirement as it was in v4.0.2:

https://github.com/lifeomic/chroma-react/blob/2c5973db869a67f1cede15ea2663f79032314dfb/src/components/TableModule/types.ts#L42-L46

I tried to fix the types in #333 but realized that it costs more to fix types than adopting a well-documented one via #332

Reply:
Gotcha. Makes sense.